lib/graphiti/resource/polymorphism.rb

```diff
@@ -42,9 +42,14 @@ def inherited(klass)
         end
 
         def sideload(name)
-          sl = super
+          if (split_on = name.to_s.split("on__")).length > 1
```

Comment:
Probably overkill, but how about splitting on a regex to limit to string start: /\Aon__/

Reply:
Done!